### ISSUE TITLE
bug: Focused input field does not update while typing.

### ISSUE THREAD

**Issue by ParitoshVaidya** [driftyco/ionic#3379]
**Type**: <span ionic-type>bug</span>

**Platform**: <span ionic-platform>android</span> <span ionic-platform-version>4.4</span>  <span ionic-webview>webview</span>

<span ionic-description>I have to click the input field once again to see the changes.

Faced this issue many times on web browser (Chrome) as well as Android 4.2, 4.4 Particularly last field on the form (on Android device). This is intermittent issue. 
</span>

<span is-issue-template></span>

**Comment by Ionitron** [comment]
Greetings @ParitoshVaidya!

My sensors indicate that you need to update your issue through our [custom issue form](http://ionicframework.com/submit-issue/#/?iid=3379). We are now requiring all issues to be submitted this way, to ensure that we have all of the information necessary to fix them as quickly as possible.

**[Click Here To Update Your Issue](http://ionicframework.com/submit-issue/#/?iid=3379)**

I will have no choice but to close this issue if it is not resubmitted through the form. Please fill out the rest of the form, so that I may use my friendly robot powers to assist you.

Thank you!

<span ionitron-issue-resubmit></span>

**Comment by ParitoshVaidya** [comment]
Having same issue on Samsung Galaxy Grand Android v4.2.2, Samsung Trend Android v4.1.2

**Comment by Ionitron** [comment]
Greetings @ParitoshVaidya!

My sensors indicate that I require more information in order to assist you.

- Can you provide a detailed description of this issue?
- How would a robot go about replicating this issue?
- On which platform does this happen occur? (iOS, Android, all, etc.)
- On which platform versions does this occur? (Android 4.2, iOS 8, all, etc.)
- Is this an issue with the Ionic Framework or Ionic CLI?

Additionally, providing a [codepen](http://codepen.io/ionic/public-list/) that replicates the issue is extremely helpful.

If you wish to get help using the framework itself, I recommended visiting [the forum](http://forum.ionicframework.com), where many humans will be able to offer wisdom and advice.

Thank you for allowing me to assist you.

<span ionitron-needs-reply></span>

**Comment by ParitoshVaidya** [comment]
Description: 

This is intermittent behavior, occurs for last field on the form. One in the following (or multiple) issue(s) occurs most of the times.

Input field does not get focus on first tap. I require to tap on it once again.
Input field does not update while typing. If tapped once again, typed text gets displayed.
Input field hides behind soft keyboard. After starting typing it becomes visible with 2nd issue.
Any idea why this is happening? I added these issues to github as well.

Also found that these issues were on github, and are closed an year before.

I am able to reproduce the issue on Moto e (1st Gen.) Android v4.4, Samsung Galaxy Grand Android v4.2.2, Samsung Trend Android v4.1.2

**Comment by remon** [comment]
I am facing the same issue for 10 days till now , no solution

**Comment by kevbaker** [comment]
Yes I am also troubleshooting this same behavior.

```
OS: Mac OS X Yosemite
Node Version: v0.12.2
Cordova CLI: 5.0.0
Ionic CLI Version: 1.3.20
Xcode version: Xcode 6.3.1 Build version 6D1002 
ios-sim version: 3.1.1 
ios-deploy version: 1.5.0
```

**Comment by asmund1** [comment]
This seems to be resolved in latest Ionic version (not Ionic CLI). Update by changing your bower.json file, update version for ionic.
"devDependencies": {
  "ionic": "driftyco/ionic-bower#1.0.0"
},
(Seems to be fixed in version 1.0.0-rc.5, in case you have trouble with newer versions.)

then run
'bower install' or 'bower update ionic'.

**Comment by perrygovier** [comment]
I'm going to mark this as resolved then.
